src/codegen/compile_classical.ml

```diff
@@ -10937,9 +10936,8 @@ and compile_prim_invocation (env : E.t) ae p es at =
 
          StackRep.of_arity return_arity,
          code1 ^^ StackRep.adjust env fun_sr SR.Vanilla ^^
+         Closure.prepare_closure_call env ^^ (* FIXME: move to front elsewhere too *)
```

Comment:
Some fields (that could have mutable portions) could be in newspace, so let's do this eagerly. TODO: spot other similar uses...

N.B. Line 10943 potentially looked into oldspace, but probably only loaded an immutable field (function index?).